apps/examples/testcase/le_tc/kernel/tc_libc_string.c

```diff
@@ -918,6 +920,95 @@ static void tc_libc_string_strtold(void)
 
 	TC_SUCCESS_RESULT();
 }
+/**
+* @fn                   :tc_libc_string_strerrorr
+* @brief                :convert the error number to string
+* @Scenario             :put error number to strerror_r and check the return which points the string value
+* API's covered         :
+* Preconditions         :none
+* Postconditions        :none
+* @return               :void
+*/
+static void tc_libc_string_strerrorr(void)
+{
+	FAR char dest_arr[BUFF_SIZE_24];
+	size_t buf_len = BUFF_SIZE_24;
```

Comment:
You don't need to declare this. You can use BUFF_SIZE_24 directly.

Reply:
modify